lib/task.js

```diff
@@ -120,7 +120,7 @@ Task.prototype.help = function (callback) {
   }
 
   log.info("help",
-    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " [flags] [task]" +
+    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " <action> <task(s)>" +
```

Comment:
Should this be `<action> <flags> <task(s)>`?

The only flag is currently `--builderrc`, but still.